gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/ZoweJwtScheme.java

```diff
@@ -99,6 +104,7 @@ public class ZoweJwtAuthCommand extends JwtCommand {
         public void apply(InstanceInfo instanceInfo) {
             if (jwt != null) {
                 final RequestContext context = RequestContext.getCurrentContext();
+                JwtCommand.setCustomHeader(context,"authorization","Bearer " + jwt);
```

Comment:
Why is is setting it twice?

Reply:
Also, use something like HttpHeaders for the name of the header, instead of the literal string

Reply:
second one in the apply block runs only when the PAT is provided as a cookie. its necessary to set them twice otherwise, authorisation header contains PAT and pass it through the south bound services.

Reply:
fixed